### DISCUSSION TITLE
Can I use color in rgba format?

### DISCUSSION THREAD

**Discussion by chanhlt12** [pymupdf/PyMuPDF#766] [Q&A] [answered]
I am adding a rectangle annot with fill and stroke colors in RGBA format. (ex: **rgba(246,0,0,0.3)**)
I couldn't find an example to use this kind of format.

I tried to use to `fitz.utils.getColor` convert the rgba to the valid format, but it does not work.

```python
import fitz

fill = fitz.utils.getColor("rgba(246 ,0, 0, 0.3)")
print(fill)
stroke = fitz.utils.getColor("rgba(100, 100, 0, 0.5)")
print(stroke)
doc = fitz.open()
page = doc.newPage()
r = fitz.Rect(10, 10, 100, 100)
annot = page.addRectAnnot(r)
annot.setColors(fill=fill, stroke=stroke)
annot.setBorder(width=1)
annot.update()
doc.save("out.pdf")
```

It would be nice if someone could share me a code snippet?

**Comment by JorjMcKie** [comment]
This is not supported by PDF itself - a restriction which has nothing to do with (Py-) MuPDF.
If you want to make your annotation transparent, just use opacity: ``annot.set_opacity(0.3)`` for a 30% opacity.

**Comment by JorjMcKie** [comment, marked as answer]
Otherwise please remember that colors in PDF are composed of floats `0 <= c <= 1` not as integers <= 255.
